### PR TITLE
Add `supports_stop` to `CoverInfo`

### PR DESCRIPTION
**Pull request by amomchilov**
Here's the corresponding change in ESP Home: https://github.com/esphome/esphome/pull/3897
Here's the corresponding change in Home Assistant to consume this value: https://github.com/home-assistant/core/pull/80104

### CODE REVIEW

File: aioesphomeapi/model.py [hunk 1]
```diff
@@ -172,7 +173,7 @@ class CoverOperation(APIIntEnum):
     IS_CLOSING = 2
 
 
-@dataclass(frozen=True)
+@dataclass(frozen=True)
```
Comment (amomchilov) [inline aioesphomeapi/model.py:176]: Oops, this was an accident. I'll amend it out

Comment (jesserockz) [reply aioesphomeapi/model.py:176]:
```suggestion
@dataclass(frozen=True)
```

**Review by jesserockz — CHANGES_REQUESTED** [review]
You need to add the new field to the api.proto and regenerate the proto files.

**Comment by amomchilov** [comment]
@jesserockz Hey Jesse.

> You need to add the new field to the api.proto and regenerate the proto files.

Done! Is there anything else I need to do here?

**Review by jesserockz — APPROVED** [review]

**Pull request merged**